SYCL/Basic/use_pinned_host_memory.cpp

```diff
@@ -1,10 +1,20 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
-// RUN: env SYCL_PI_TRACE=2 %CPU_RUN_PLACEHOLDER %t.out 2>&1 %CPU_CHECK_PLACEHOLDER
+// RUN: %clangxx %s -o %t1.out -lsycl -I %sycl_include
+// RUN: %RUN_ON_HOST %t1.out
```

Comment:
```suggestion
// RUN: %HOST_RUN_PLACEHOLDER %t1.out
```
Different substitution is used in the llvm-test-suite